task2/task2_solution.py

```diff
@@ -0,0 +1,110 @@
+# NOTES:
+#
+# no idea whether it's ok to make
+# to_letters function a method of the class
+# (staticmethod doesn't work)
+# external func works fine, let is be so
+#
+# besides that, Journal class feels to be
+# not really universal (adding more fields to
+# input data requires changes in code)
+
+
+def to_letters(grades):
```

Comment:
Please move it into Jornal class
It should accept self as first parameter `def to_letters(self, grades):` and should be called like `self.to_letters(grades)`

Reply:
There is no need to dive into static methods or classmethods
Make it as simple as possible for now